handlers/admin_handler.py

```diff
@@ -0,0 +1,129 @@
+from bottypes.command import *
+from bottypes.command_descriptor import *
+from bottypes.invalid_command import *
+from handlers.handler_factory import *
+from handlers.base_handler import *
+from addons.syscalls.syscallinfo import *
+import re
+
+class ShowAdminsCommand(Command):
+    """Shows list of users in the admin user group."""
+    
+    def execute(self, slack_wrapper, args, channel_id, user_id):
+        """Execute the ShowAdmins command."""
+
+        admin_users = HandlerFactory.botserver.get_config_option("admin_users")
+
+        if admin_users:
+            response = "Administrators\n"
+            response += "===================================\n"
+
+            for admin_id in admin_users:
+                user_object = slack_wrapper.get_member(admin_id)
+            
+                if user_object['ok']:
+                    response += "*{0}* ({1})\n".format(user_object['user']['name'], admin_id)
+
+            response += "==================================="
+
+            response = response.strip()
+
+            if response == "":  # Response is empty
+                response += "*No entries found*"
+        
+            AdminHandler.send_message(slack_wrapper, channel_id, user_id, response)
+        else:
+            response = "No admin_users group found. Please check your configuration."
+
+            AdminHandler.send_message(slack_wrapper, channel_id, user_id, response)
+
+class AddAdminCommand(Command):
+    """Add an user to the admin user group."""
+    
+    def execute(self, slack_wrapper, args, channel_id, user_id):
+        """Execute the AddAdmin command."""
+        user = args[0].upper()
+
+        if (user.startswith("<@")) and (user.endswith(">")):
```

Comment:
Don't see the outer parens

Reply:
We do this elsewhere. Maybe add a util to get a user object from a username or @username.